examples/cluster-with-cni/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "metal_project_id" {
+  description = "Equinix project ID"
+  sensitive   = true
+  type        = string
+}
+
+variable "metal_auth_token" {
+  description = "Equinix provider user auth token"
+  sensitive   = true
+  type        = string
+}
+
+variable "metro" {
+  description = "Metro of choice"
+  type        = string
+  default     = "da"
+}
+
+variable "kube_vip_version" {
+  description = "KubeVip version of choice"
+  type        = string
+  default     = "v0.6.2"
+}
+
+variable "kubernetes_version" {
+  description = "Kubernetes version. See https://www.downloadkubernetes.com/"
```

Comment:
Is this an official URL? Maybe we should refer to this:
https://kubernetes.io/releases/patch-releases/#detailed-release-history-for-active-branches